data/releases/schedule.yaml

```diff
@@ -74,10 +105,20 @@ schedules:
   maintenanceModeStartDate: 2022-08-28
   endOfLifeDate: 2022-10-28
   next:
-    release: 1.22.14
-    cherryPickDeadline: 2022-09-09
-    targetDate: 2022-09-14
+    release: 1.22.15
+    cherryPickDeadline: 2022-09-20
+    targetDate: 2022-09-21
```

Comment:
I'm not sure if this is correct. 1.22.15 is supposed to be the final 1.22 release, but 1.22 is still not EOL so we can't remove this. I think this might be the best approach.

Reply:
we can release this, it is not eol and till there we do patch releases for urgent/critical things

Reply:
Should I put October as the target date then?

Reply:
The `next` section supports a note. We can add a note here that the patch release will be done if and only if there are CVE/critical dependency updates.

Reply:
Thanks for letting me know! I updated the PR to add a note. PTAL if it's okay.